aspnetcore/introduction-to-aspnet-core.md

```diff
@@ -68,36 +149,34 @@ We're working hard to close the API gap from .NET Framework to .NET Core. The [W
 
 We recommend the following sequence of tutorials and articles for an introduction to developing ASP.NET Core apps:
 
-1. Follow a tutorial for the type of app you want to develop or maintain:
+1. Follow a tutorial for the type of app you want to develop or maintain.
 
    |App type  |Scenario  |Tutorial  |
    |----------|----------|----------|
    |Web app                   | For new development        |[Get started with Razor Pages](xref:tutorials/razor-pages/razor-pages-start) |
    |Web app                   | For maintaining an MVC app |[Get started with MVC](xref:tutorials/first-mvc-app/start-mvc)|
    |Web API                   |                            |[Create a web API](xref:tutorials/first-web-api)\*  |
    |Real-time app             |                            |[Get started with SignalR](xref:tutorials/signalr) |
-   |Blazor app                |                            |[Get started with Blazor](xref:blazor/get-started) |
-   |Remote Procedure Call app |                            |[Get started with a gRPC service](xref:tutorials/grpc/grpc-start) |
 
-1. Follow a tutorial that shows how to do basic data access:
+1. Follow a tutorial that shows how to do basic data access.
 
    |Scenario  |Tutorial  |
    |----------|----------|
    | For new development        |[Razor Pages with Entity Framework Core](xref:data/ef-rp/intro) |
-   | For maintaining an MVC app |[MVC with Entity Framework Core](xref:data/ef-mvc/intro)
-
-1. Read an overview of ASP.NET Core features that apply to all app types:
+   | For maintaining an MVC app |[MVC with Entity Framework Core](xref:data/ef-mvc/intro) |
 
-   * [Fundamentals](xref:fundamentals/index)
+1. Read an overview of ASP.NET Core [fundamentals](xref:fundamentals/index) that apply to all app types.
 
 1. Browse the Table of Contents for other topics of interest.
 
-\* There is a new [web API tutorial that you follow entirely in the browser](https://docs.microsoft.com/learn/modules/build-web-api-net-core), no local IDE installation required.  The code runs in an [Azure Cloud Shell](https://azure.microsoft.com/features/cloud-shell/), and [curl](https://curl.haxx.se/) is used for testing.
+\* There is a [web API tutorial that you follow entirely in the browser](/learn/modules/build-web-api-net-core), no local IDE installation required. The code runs in an [Azure Cloud Shell](https://azure.microsoft.com/features/cloud-shell/), and [curl](https://curl.haxx.se/) is used for testing.
```

Comment:
If daggerizing ... and the following matches the "There's also a ..." structure used earlier ...

```suggestion
&dagger;There's also a [web API tutorial that you follow entirely in the browser](/learn/modules/build-web-api-net-core), no local IDE installation required. The code runs in an [Azure Cloud Shell](https://azure.microsoft.com/features/cloud-shell/), and [curl](https://curl.haxx.se/) is used for testing.
```